app/about_us/ourCrew.js

```diff
@@ -0,0 +1,25 @@
+import styles from "../../components/cardOurCrew/cardOurCrew.module.css"
+import OurCrewData from "@/data/ourCrewData";
+import OurCrewCard from "@/components/cardOurCrew/ourCrewCard";
+
+
+export default function OurCrew() { 
+   return <>
+      <p className={styles.ourCrewSubtitle}>Our crew</p>
+      <h2 className={styles.ourCrewTitle}>Who we are</h2>
+      <p className={styles.ourCrewText}>Our crew is the heart and soul of Galactica. We are a diverse team of seasoned space explorers, engineers, and visionaries who are united by a common goal: to make space travel accessible and exciting for all.</p>
+      <div className={styles.ourCrewCards}>
+         {OurCrewData.map((card, index) => (
```

Comment:
Very nice!

Using the `.map()` function was not a part of this weeks HW, so great that you are already using it and practicing 💪